libs/shared/src/entities/message-template/message-template.interface.ts

```diff
@@ -9,13 +9,21 @@ export interface IEmailBlock {
   };
 }
 
+export class ITemplateVariable {
+  type: 'String' | 'Array' | 'Boolean';
```

Comment:
Maybe we can create a type out of it? 

```
type TemplateVariableType = 'String' | 'Array' | 'Boolean';
```

Or even an enum. And we can have it in the @novu/shared to be reused between client and backend